### PR TITLE
Changed How To String to FAQ and added Discuss Tab to the Navigation bar

### PR DESCRIPTION
**Pull request by thinksanky**
- Provide a Discuss Navigation tabbed link on the top navigation bar.
- Changed How To to FAQ.

### CODE REVIEW

**Comment by kevinthesun** [comment]
Modify navbar also requires changes in https://github.com/apache/incubator-mxnet/blob/master/docs/_static/js/navbar.js#L3 and https://github.com/apache/incubator-mxnet/blob/master/docs/_static/js/sidebar.js#L3. Otherwise it'll break some functions.

File: docs/_static/js/sidebar.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /*Preprocess*/
 var LANG = ['python', 'scala', 'r', 'julia', 'c++', 'perl'];
-var TITLE_WITH_LANG = ['/get_started/', '/tutorials/', '/how_to/', '/architecture/', '/community/'];
+var TITLE_WITH_LANG = ['/get_started/', '/tutorials/', '/FAQ/', '/architecture/', '/community/'];
```
Comment (kevinthesun) [inline docs/_static/js/sidebar.js]: Same here.

File: docs/_static/js/navbar.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 var searchBox = $("#search-input-wrap");
 var TITLE = ['/get_started/', '/tutorials/', '/community/contribute.html', '/api/'];
-var DOC_TITLE = ['/how_to/', '/architecture/', '/model_zoo/'];
+var DOC_TITLE = ['/FAQ/', '/architecture/', '/model_zoo/'];
```
[the remark this answers is not in the document]
Comment (thinksanky) [reply docs/_static/js/navbar.js]: Got it! did not release it was the URL. Was thinking about the folder naming. Fixed the code now.

**Comment by kevinthesun** [comment]
LGTM

**Pull request merged**